core/common/src/main/scala/org/typelevel/otel4s/baggage/BaggageManager.scala

```diff
@@ -38,3 +52,7 @@ trait BaggageManager[F[_]] extends Local[F, Baggage] {
   def getValue(key: String): F[Option[String]] =
     reader(_.get(key).map(_.value))
 }
+
+object BaggageManager {
+  def apply[F[_]](implicit ev: BaggageManager[F]): BaggageManager[F] = ev
```

Comment:
it's currently only provided explicitly in `Otel4s`; should we change it to an `implicit def` (like `localContext`) there?

Reply:
It's in the same boat as `MeterProvider` and `TracerProvider`. I don't think we should define them as `implicit def`.